.github/workflows/cibuild.yml

```diff
@@ -53,9 +50,31 @@ jobs:
       with:
         java-version: ${{ matrix.java }}
     - name: Build
-      run: |
-        ./.github/scripts/${{ matrix.script }}
+       # Can't upgrade this until https://github.com/GabrielBB/xvfb-action/issues/10 is fixed
+      uses: GabrielBB/xvfb-action@v1.0
```

Comment:
I am not keen on using this action and these changed to the workflow. See https://github.com/bndtools/bnd/pull/4208 for my suggested alternate.

Reply:
I have reviewed and approved your alternate solution - agreed, it is better. I didn't realise that xvfb was part of the standard GitHub Actions offering.

I have also rebased this PR onto yours (minus the commit that changes the workflow).